source/views/menus/components/filter-menu-toolbar.js

```diff
@@ -13,40 +14,54 @@ const styles = StyleSheet.create({
 	},
 	toolbarSection: {
 		flexDirection: 'row',
+		alignItems: 'center',
 	},
 })
 
 type PropsType = {
 	date: momentT,
 	title?: string,
-	onPress: () => any,
+	onPopoverDismiss: (filter: FilterType) => any,
```

Comment:
Why are we only returning the changed filter (singular) here? Why not return all of the filters? Wouldn't that simplify usage when we use this component (being able to just replace our `state.filters` instead of update it)?

Reply:
We're not *returning* only the changed filter, we're *passing* it upstream through its ancestor components until it eventually gets to the component that holds the filters in its state (`FancyMenu`, in this case). The reason I did it this way is so that I could pass the same `onPopoverDismiss` prop to the `FilterToolbarButton` and the `FilterToolbar`, within the `FilterMenuToolbar` component. I guess what I'm trying to say is that the individual toolbar buttons should only be aware of the filter they control, while their parents should know about all the other filters and take care of things when one of their children (toolbar buttons) informs them that they have changed. Idk does that make sense?